auth-server-open-api-spec.yaml

```diff
@@ -429,23 +429,11 @@ components:
         - $ref: '#/components/schemas/access-incoming'
         - $ref: '#/components/schemas/access-outgoing'
       description: The access associated with the access token is described using objects that each contain multiple dimensions of access.
-    limits:
-      title: limits
-      description: Open Payments specific property that defines the limits under which outgoing payments can be created.
+    limits-outgoing:
+      title: limits-outgoing
+      description: Open Payments specific property that defines the limits under which outgoing payments can be created. Either `sendAmount` or `receiveAmount` is REQUIRED.
       type: object
       properties:
-        startsAt:
-          type: string
-          format: date-time
-          description: ISO 8601 date-time indicating when the grant becomes effective.
-        expiresAt:
-          type: string
-          format: date-time
-          description: ISO 8601 date-time indicating when the grant expires.
-        interval:
-          type: string
-          format: duration
-          description: 'ISO 8601 duration indicating how often the amounts, `sendAmount` or `receiveAmount`, are available.'
         receivingAccount:
```

Comment:
The descriptions for `receivingAccount` and `receivingPayment` say that one is required. Is that still the case?

Reply:
If there is neither, it means I can just create outgoing payments to whoever using that grant. Sounds like we need that for web monetization. So they should both be optional.